Code/GraphMol/ForceFieldHelpers/catch_tests.cpp

```diff
@@ -0,0 +1,54 @@
+
+//
+//  Copyright (C) 2024 Niels Maeder and other RDKit contributors
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include <cmath>
+#include <RDGeneral/test.h>
+#include <catch2/catch_all.hpp>
+
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <ForceField/MMFF/Params.h>
+#include <ForceField/MMFF/BondStretch.h>
+
+#include "FFConvenience.h"
+
+using namespace RDKit;
+
+TEST_CASE("Test empty force field") {
+  auto mol =
+      "CCCO |(-1.28533,-0.0567758,0.434662;-0.175447,0.695786,-0.299881;0.918409,-0.342619,-0.555572;1.30936,-0.801512,0.71705)|"_smiles;
+  REQUIRE(mol);
+  SECTION("basics") {
+    auto forceField = ForceFieldsHelper::createEmptyForceFieldForMol(*mol);
+    REQUIRE(forceField);
+    forceField->initialize();
+    TEST_ASSERT(forceField->minimize() == 0);
```

Comment:
Sorry I'm just noticing this on the second review.
You shouldn't be using `TEST_ASSERT()` here, that's something we use in the old RDKit tests.
When you are using the catch tests, the correct way to test something is `CHECK` if it's ok to continue if the test fails or `REQUIRE` if it's not ok.
The options are enumerated here:
https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md